src/nicypkg/functions.nim

```diff
@@ -73,8 +63,8 @@ proc color*(s: string, fg: string = "", bg: string = "",
     result = reverse(result)
   result = reset(result)
 
-proc horizontalRule*(c: char = '-'): string =
-  for _ in 1 || terminalWidth(): # Parallel for loop, unordered iter but faster.
+proc horizontalRule*(c = '-'): string =
+  for _ in 1 .. terminalWidth():
```

Comment:
The || loop here was REALLY unnecessary - all we do is just adding some characters, OpenMP loop might break stuff and it'll almost surely be slower. The only reason why it worked is because we compiled without threads or OpenMP.